c/zss.c

```diff
@@ -1226,6 +1266,8 @@ int main(int argc, char **argv){
       installDatasetMetadataService(server);
       installDatasetContentsService(server);
       installDatasetEnqueueService(server);
+      installDatasetHeartbeatService(server);
+      initDatasetLocking(server);
```

Comment:
This doesn't belong in zss.c, it's specific to the dataset services. All ZSS needs to care about here is the install functions.

Reply:
moved inside datsetService.c